### PR TITLE
GT-449 Update Features table in doc

### PR DESCRIPTION
**Pull request by jwierzbo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,30 +46,31 @@ covers individual newer features separately.
...
+| Spec Default Restore                    | 1.2.21           | >= 3.7.0         | Community, Enterprise | 1.2.21     | Beta         | True    | --deployment.feature.deployment-spec-defaults-restore | If set to False Operator will not change ArangoDeployment Spec                                                   |
+| Force Rebuild Out Synced Shards         | 1.2.27           | >= 3.8.0         | Community, Enterprise | 1.2.27     | Beta         | False   | --deployment.feature.force-rebuild-out-synced-shards  | It should be used only if user is aware of the risks. [doc](./docs/design/features/rebuild_out_synced_shards.md) |
+| Failover Leader service                 | 1.2.13           | >= 3.7.0         | Community, Enterprise | 1.2.13     | Production   | False   | --deployment.feature.failover-leadership              | [doc](./docs/design/features/failover_leader_service.md)                                                         |
```
Comment (nikita-vanyasin) [inline README.md:73]:
```suggestion
| Failover Leader service                 | 1.2.31           | >= 3.7.0         | Community, Enterprise | 1.2.13     | Production   | False   | --deployment.feature.failover-leadership              | [doc](./docs/design/features/failover_leader_service.md)                                                         |
```

[the remark this answers is not in the document]
Comment (jwierzbo) [reply README.md:73]: @nikita-vanyasin that feature has been provided a long time ago. I'm just adding missing doc

[the remark this answers is not in the document]
Comment (nikita-vanyasin) [reply README.md:73]: Ah I see! Just didn't expect the old feature to be in the end of the list. Sorting by operator version make sense IMHO 👌🏻

[the remark this answers is not in the document]
Comment (jwierzbo) [reply README.md]: @nikita-vanyasin done

**Review by informalict — APPROVED** [review]
lgtm

**Pull request merged**